terraform_compliance/steps/steps.py

```diff
@@ -158,10 +158,10 @@ def i_have_name_section_configured(_step_obj, name, type_name='resource', _terra
     skip_step(_step_obj, name)
 
 
-@when(u'its {key:PROPERTY} is {value:PROPERTY}')
-@when(u'its {key:PROPERTY} has {value:PROPERTY}')
-@when(u'its {key:PROPERTY} includes {value:PROPERTY}')
-@when(u'its {key:PROPERTY} contains {value:PROPERTY}')
+@when(u'its {key:PROPERTY} is {value:ANY}')
+@when(u'its {key:PROPERTY} has {value:ANY}')
+@when(u'its {key:PROPERTY} includes {value:ANY}')
+@when(u'its {key:PROPERTY} contains {value:ANY}')
```

Comment:
It looks like this causes a problem and messes up with `its {key:PROPERTY} is not {value:PROPERTY}` test. Because `not {value:PROPERTY}` is matched by `{value:ANY}`.

I recommend to add a new step definition there like ;

```python
@when(u'its {key:PROPERTY} contains "{value:ANY}"')
```

of course same should be added to negative option too like ;

```python
@when(u'its {key:PROPERTY} contains "{value:ANY}"')
```

Reply:
I hope this is what you had in mind.
I think there a other code issues around the the negative options. I'll address those in a separate PR.